src/arch/dotproductavx.cpp

```diff
@@ -29,6 +29,47 @@ namespace tesseract {
 
 // Computes and returns the dot product of the n-vectors u and v.
 // Uses Intel AVX intrinsics to access the SIMD instruction set.
+#if defined(FAST_FLOAT)
+float DotProductAVX(const float *u, const float *v, int n) {
```

Comment:
FAST_FLOAT16 -> FAST_FLOAT ?

Not just in this line, but also in other places in this file.

Reply:
I used `FAST_FLOAT16` up to now exclusively in `dotproductavx.cpp` to disable some code which handled 16 (instead of 8) float values per iteration and which had a negative impact on the performance.